.github/workflows/helm_release.yaml

```diff
@@ -1,10 +1,8 @@
+# This workflow releases new version of helm charts
 name: Release Charts
 
 on:
   workflow_dispatch:
-  push:
```

Comment:
Thanks, it was also manually disabled but it is better to remove this.

Btw by default it shouldn't do a release it should verify if there is a new `version` in `chart/Chart.yaml` file and release only a new version.